src/components/home/mainText.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const MainText = () => {
+  return (
+    <div className="flex items-center justify-center font-seasons text-[6vw]">
```

Comment:
```suggestion
    <div className="flex justify-center font-seasons text-[6vw]">
```
remove items-center it won't do anything here